spec/features/visit_product_page_spec.rb

```diff
@@ -16,7 +16,7 @@
     tag_component = PiwikTagComponent.new
     expect(tag_component).to be_in_html
     expect(tag_component.to_s).to include 'setEcommerceView'
-    expect(tag_component.to_s).to eq fixture('product_example_com.html')
+    expect(tag_component.as_ecma).to eq fixture('product_example_com.js')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.